{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -47,14 +47,15 @@ classifiers = [
 {%- if cookiecutter.backend == "hatch" %}
 dynamic = ["version"]
 {%- endif %}
-dependencies = ["pybamm"]
+dependencies = ["pybamm",]
 
 [project.optional-dependencies]
 dev = [
   "pytest >=6",
   "pytest-cov >=3",
-  "nox",
+  "nox[uv]",
   "pre-commit",
+  "pytest-cookies",
```

Comment:
```suggestion
```

I don't think the generated project will need `pytest-cookies` as a dependency, because the generated project won't have a template to provide to users (and they won't need to test the generation of a project).